.github/workflows/ci.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
     - uses: ruby/setup-ruby@v1
       with:
         bundler-cache: true
+    - run: make ruby-audit
```

Comment:
So are we going to remove this step?

Reply:
No. Fixing vulnerabilities in gems are still needed.

Reply:
Yes, but I believe it should not block ci because it is development dependencies?
Anyway let me merge this first.

Reply:
It will be very difficult to separate the development dependencies that might generate code that to be used in production. for example program used in `go generate` might have CVE that generate faulty code. So from compliance perspective, we will cover the dev dependencies.

The same logic follow, developer machine is in the compliance coverage too.